ofl/justfonthuninn/METADATA.pb

```diff
@@ -0,0 +1,25 @@
+name: "Justfont Huninn"
+designer: "Justfont"
```

Comment:
I'm not sure if we can include "Justfont" in the family name. Our guide seems to prohibit it, https://googlefonts.github.io/gf-guide/onboarding.html/

cc @davelab6

Reply:
If it is ok for “IBM Plex” it should be OK here, no?

Reply:
I hear what you're saying but I think IBM Plex was introduced before the rule. Thoughts @davelab6

Reply:
Hey @m4rc1e! The name is still in discussion here: https://github.com/google/fonts/issues/7740#issuecomment-2755658616 We suggested to remove 'Justdont'

Reply:
Font name updated!